kyhorne/booksrus
# User authentication

References #19 

Provides user authentication for bookstore owner or User. Set up with one user and one admin with credentials "user","user" and "admin","admin".

To test pull request:
Attempt to reach endpoints without authenticating
Ensure admin can upload and user cannot

Clearing the browser's cache is the easiest way to switch users
